src/shared/theme/theme.ts

```
@@ -1,21 +1,314 @@
+              '0 3px 5px -1px rgba(0, 0, 0, 0.2), 0 6px 10px 0 rgba(0, 0, 0, 0.14), 0 1px 18px 0 rgba(0, 0, 0, 0.12)',
+            background: customPalette.light[200],
+          },
+          '&.Mui-disabled': {
+            background: `#9cc2bd`,
+            color: customPalette.gray[900],
+          },
+        }),
+        sizeLarge: () => ({
+          textTransform: 'capitalize',
+        }),
+        startIcon: ({}) => ({
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)